### PR TITLE
Update core to version 25955fc5

### PR DESCRIPTION
**Pull request by hculea**

### CODE REVIEW

File: example/main.go [hunk 1]
```diff
@@ -31,6 +30,7 @@ func main() {
 	item := createAndGetItem(client)
 	getAndUpdateItem(client, item.VaultID, item.ID)
 	listVaultsAndItems(client, item.VaultID)
+	validateSecretReference(item.VaultID,item.ID,"username")
```
Comment (hculea) [inline example/main.go:33]: Let's try to format this nicer:
```suggestion
	validateSecretReference(item.VaultID, item.ID, "username")
```

File: internal/release/RELEASE-NOTES [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
 
-The 1Password Go SDK allows you to build secrets management integrations that can programmatically access secrets stored in 1Password. With this SDK, you can:
-
-- [Securely load secrets](https://developer.1password.com/docs/sdks/load-secrets/) from 1Password into your code with secret references.
-- [Read, write, and update secrets](https://developer.1password.com/docs/sdks/manage-items/) stored in 1Password, including passwords, API keys, and one-time passwords.
-- [List items and vaults](https://developer.1password.com/docs/sdks/list-vaults-items/) in a 1Password account.
+This release includes:
+   * introduces support for tags, users can now create, update, and delete tags on 1Password Items.
+   * introduces a `Validate Secret Reference` function that will ensure the syntax of your secret reference is error free.
+   * Fields and Sections are now optional in `ItemCreateParams` and the user won't have to specify them if not needed anymore.
```
Comment (hculea) [inline internal/release/RELEASE-NOTES:6]: This is not actually the case for Go, the fields were already optional here.

File: internal/release/RELEASE-NOTES [hunk 3]
```diff
@@ -1,7 +1,6 @@
-This is the v0 release of the 1Password Go SDK.
```
Comment (hculea) [inline internal/release/RELEASE-NOTES:1]: note for reviewer - ignore the changes in this file, we're deciding about the content here separately.

**Comment by hculea** [comment]
Functionally tested and it works great! I've just pushed the updated release notes and updated example.

File: example/main.go [hunk 4]
```diff
@@ -111,6 +113,27 @@ func resolveSecretReference(client *onepassword.Client, vaultID, itemID, fieldID
...
+}
+
+func validateSecretReference(vaultID, itemID, fieldID string) {
+	// [developer-docs.sdk.go.validate-secret-reference]-start
+	// Validate your secret reference
+	err := onepassword.ValidateSecretReference(context.Background(), fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID))
+	if err != nil {
+		panic(err)
+	}
+	// [developer-docs.sdk.go.validate-secret-reference]-end
```
Comment (AndyTitu) [inline example/main.go:134]: What does this comment represent?

Comment (AndyTitu) [reply example/main.go:134]: Hmm, I think I found the answer: this is for the new docs portal reference syncer tool, right?

**Review by AndyTitu — APPROVED** [review]

**Pull request merged**